hadoop-cloud/src/hadoop-3/test/scala/org/apache/spark/internal/io/cloud/CommitterBindingSuite.scala

```diff
@@ -76,6 +78,16 @@ class CommitterBindingSuite extends SparkFunSuite {
     assert(inner.jobCommitted, s"$inner job not committed")
     parquet.abortJob(tContext, JobStatus.State.RUNNING)
     assert(inner.jobAborted, s"$inner job not aborted")
+
+    val binding = new BindingPathOutputCommitter(path, tContext)
+    if (binding.isInstanceOf[StreamCapabilities]) {
```

Comment:
Now the test is in the `hadoop-3` directory so do we still need this `if`?

Reply:
yes, because the new feature isn't in hadoop 3.3.4 or earlier....this test suite works with the current and future hadoop releases. the new feature needs the forthcoming hadoop branch-3.3. release

Reply:
I see.